### PR TITLE
UI support for verbose analyzer output

### PR DESCRIPTION
**Pull request by jkppr**
+ UI rendering of the new verbose analyzer output introduced in #2706 
+ Using the `LinkRedirectWarn.vue` component for the reference links.
+ Support for connected events
  + Saved searches, Stories, Graphs and Tags that are added during the analyzer run and are linked in the output object will now be linked. 

TS-verbose-analyzer-results

### CODE REVIEW

File: timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue [hunk 1]
```diff
@@ -81,33 +97,173 @@ limitations under the License.
...
+              </td>
+              <td style="border: none">
+                <span>
+                  {{ verboseAnalyzerOutput.result_status }}
+                </span>
+              </td>
+            </tr>
+            <tr v-if="Object.keys(getAnalyzerOutputMetaData).length !== 0">
+              <td colspan="2" style="border: none">
+                <strong>Connected Events</strong>
```
Comment (berggren) [inline timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue:179]: As discussed I would like to find some other wording here. `Events` is pretty broad and overloaded term.

Some ideas
- Supportive output
- Analyzer Output
- Result artifacts
- Explore output

File: timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue [hunk 2]
```diff
@@ -81,33 +97,173 @@ limitations under the License.
...
+                  : 'dark-hover'
+                : expanded
+                ? 'light-hover light-bg'
+                : 'light-hover'
+            "
+          >
+            <tr class="pr-3">
+              <td width="80" style="border: none">
+                <strong v-if="timeline.analysis_status === 'ERROR'">Error:</strong>
+                <strong v-else>Result:</strong>
```
Comment (berggren) [inline timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue:251]: Maybe call this `Verdict` instead?

File: timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue [hunk 3]
```diff
@@ -123,21 +279,148 @@ limitations under the License.
...
+            let view = this.meta.views.find((view) => view.id === id)
+            if (view !== undefined) {
+              metaData['Searches'].push(view)
+            }
+          }
+        }
+        if (this.verboseAnalyzerOutput.platform_meta_data.saved_stories !== undefined) {
+          metaData['Stories'] = []
+          for (const id of this.verboseAnalyzerOutput.platform_meta_data.saved_stories) {
+            let storie = this.meta.stories.find((storie) => storie.id === id)
```
Comment (berggren) [inline timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue:338]: Spelling: `story` in singular

**Review by berggren — APPROVED** [review]
Overall looks good (approved). Some wording that we discussed offline, and a spelling nit.
Feel free to merge as you see fit.

File: timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue (hunk 1 above)
Comment (jkppr) [reply timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue:179]: Thanks for the great suggestions. I like `Result artifacts` and will change it.

File: timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue (hunk 2 above)
Comment (jkppr) [reply timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue:251]: I think I will go with `Summary` first and collect user feedback on the chosen wording.

**Pull request merged**